README.md

```diff
@@ -39,39 +39,82 @@ We can't wait to see the community you selfhost with Forem!
 
 _Note: Following this quick start guide with the cloud provider of your choice will cost you money! Please consult with each cloud provider to figure out how much your Forem will cost you per month._
 
-1) Clone the forem/selfhost repo to your local computer:
-     - `git clone https://github.com/forem/selfhost.git`
-2) Change into the selfhost directory: `cd selfhost`
-3) Install Requirements:
-    - `pip3 install -r requirements.txt`
-4) Generate an Ansible Vault password
-    - `pwgen -1 24|tee ~/.forem_selfhost_ansible_vault_password`
-5) Copy example Ansible Inventory from `inventory/example/setup.yml` to `inventory/forem/setup.yml`
-6) Edit `inventory/forem/setup.yml` Ansible Inventory with your Forem settings
+Start by cloning the `forem/selfhost` repository to your local computer and change into it:
+
+```
+git clone https://github.com/forem/selfhost.git
+cd selfhost
+```
+
+After this step you have two choices: a semi-automated setup via a script or a completely manual installation.
+
+### Semi-automated setup
+
+We have a script in place that can provide several of the necessary setup tasks for you. It will perform steps the first 3 steps of the manual installation process (installing Python dependencies, generating an Ansible Vault password and copying the inventory definition for you). It will also generate the secrets needed for step 4.
+
+```
+./setup
+
+Verifying that pip is available
+
+Installing Python dependencies
+[output omitted]
+
+Generating Ansible Vault secret
+ixooGe3ob0shob8soo6AhYie
+
+Copying example inventory
+'inventory/example/setup.yml' -> 'inventory/forem/setup.yml'
+
+Generating Vault secrets
+[output omitted]
+
+Use these secrets to replace the placeholders in inventory/forem/setup.yml
+```
+
+Once the script finished running, continue from step 4. of the manual installation process described below.
+
+### Manual installation
+
+1) Install Python dependencies:
```

Comment:
I changed all the numbers to `1` to make use of Markdown's automatic numbering feature.